include/z64.h

```diff
@@ -184,10 +184,22 @@ typedef struct {
     /* 0x0104 */ Vec3f  curDistortionOrientation;
     /* 0x0110 */ Vec3f  curDistortionScale;
     /* 0x011C */ u16    normal; // used to normalize the projection matrix
-    /* 0x0120 */ s32    flags;
+    /* 0x0120 */ s32    dirty;
     /* 0x0124 */ s32    unk_124;
 } View; // size = 0x128
 
+#define VIEW_VIEWING 0x1
+#define VIEW_VIEWPORT 0x2
+#define VIEW_PROJECTION_PERSPECTIVE 0x4
+#define VIEW_PROJECTION_ORTHOGRAPHIC 0x8
+#define VIEW_ALL 0xF
```

Comment:
```suggestion
#define VIEW_ALL (VIEW_VIEWING | VIEW_VIEWPORT | VIEW_PROJECTION_PERSPECTIVE | VIEW_PROJECTION_ORTHOGRAPHIC)
```
`VIEW_PROJECTION_ORTHOGRAPHIC` could possibly be worth shortening, since we use `ortho` everywhere else currently. Less sure but `VIEW_PROJECTION_PERSPECTIVE` could be shortened to `persp` possibly, although that's used less than `ortho` is.

Reply:
Shortened 273f422f2f915be7d78d40f3897aa555345c011f

Reply:
As for the `|`, not sure how strong the suggestion is intended to be, and if it's project policy I will of course just change it, but I personally find the `0xF` version a lot more readable, whereas the other way just starts to look like a fatiguing block of capital letters. It's already called `ALL` and there can only ever be four flags without rewriting the whole system so errors there seem unlikely.

Reply:
seeing that its all of the flags being set together is much more important information to get at a glance, than just seeing some magic number being set. 

this is also why tharos suggestion is pretty important with the other flags being set as a shift of the others. The relationship between these flags is important and must be reflected in the define, over just being a bunch of hex numbers

Reply:
35f4fae87